tests/phpunit/tests/formatting/emoji.php

```diff
@@ -122,15 +122,15 @@ public function test_wp_emoji_list_returns_data() {
 		$entities = _wp_emoji_list( 'entities' );
 		$this->assertNotEmpty( $entities, 'Entities should not be empty' );
 		$this->assertIsArray( $entities, 'Entities should be an array' );
-		// Emoji 15 contains 3718 entities, this number will only increase.
-		$this->assertGreaterThanOrEqual( 3718, count( $entities ), 'Entities should contain at least 3718 items' );
+		// Emoji 17 contains 4007 entities, this number will only increase.
+		$this->assertGreaterThanOrEqual( 4007, count( $entities ), 'Entities should contain at least 4007 items' );
```

Comment:
```suggestion
		$this->assertEquals( 4007, count( $entities ), 'Emoji 17 contains 4,007 entities.' );
```
Should we change this to confirm the exact number? This would ensure the test is updated with each update to the library introducing support for the next version of Emoji.

Reply:
I think the purpose is clearer with >= as it's possible to tell at a glance that the number should never go down. If something goes wrong/pagination rules change on the GitHub API we don't want to simply update it to, say, 500.

Reply:
That works for me, @peterwilsoncc.